tests/utils.ts

```diff
@@ -204,6 +204,13 @@ export async function dragComponent(
       steps,
     }
   );
+  await page.mouse.move(
```

Comment:
Add a comment linking to the playwright docs here: https://playwright.dev/docs/input#dragging-manually